### PR TITLE
Add myNode as connector

### LINKED ISSUE #676: Add myNode as connector option
We would like to add as many plug and play node connectors as possible to make it easy for users.

### PR DESCRIPTION
**Pull request by secondl1ght**
#### Link this PR to an issue
Fixes #676 

#### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

#### Describe the changes you have made in this PR -
I started adding in the myNode connector, I used @bumi's LNDConnect code from the Umbrel connector because myNode also has LNDConnect capabilities. I have the front-end setup however I am testing this locally and cannot get it connected for some reason. So I am leaving this PR as a draft for now. Maybe @bumi we could debug together sometime?

#### Screenshots of the changes (If any) -
image

image

#### How Has This Been Tested?
Not working yet - have not been able to connect.

#### Checklist:

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I have made corresponding changes to the documentation (If Any)

### CODE REVIEW

**Comment by bumi** [comment]
nice! this is great! 
let's debug your issue. what error do you see? do you have the native companion installed? (I think we miss a Linux installer there - wanna help with that?)

File: src/app/screens/Onboard/ChooseConnector/index.tsx [hunk 1]
```diff
@@ -63,7 +70,7 @@ export default function ChooseConnector({ title, description }: Props) {
             </p>
           )}
         </div>
-        <div className="grid grid-cols-5 gap-5">
+        <div className="grid grid-cols-4 gap-5">
```
Comment (bumi) [inline src/app/screens/Onboard/ChooseConnector/index.tsx:73]: why did you change it to be 4 columns? Alby should be in the first line - it is also the easiest to get started.

Comment (secondl1ght) [reply src/app/screens/Onboard/ChooseConnector/index.tsx:73]: We used to have it at 4 columns, I thought we might want to change it back? If not I can remove this.

**Comment by secondl1ght** [comment]
> nice! this is great! let's debug your issue. what error do you see? do you have the native companion installed? (I think we miss a Linux installer there - wanna help with that?)

Yes I would like to make it easier to work for Linux, how can we get started with this?

I'll open this up for review if anyone else is running myNode they can test it. It should work, I think my issues might be local because I cannot access the IP:port locally which i'm trying to get working.

File: src/app/screens/Onboard/ChooseConnector/index.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import lndhub from "/static/assets/icons/lndhub.png";
 import lnd from "/static/assets/icons/lnd.png";
 import eclair from "/static/assets/icons/eclair.jpg";
 import alby from "/static/assets/icons/alby.png";
-import umbrel from "/static/assets/icons/umbrel.png";
+import mynode from "/static/assets/icons/mynode.png";
```
Comment (bumi) [inline src/app/screens/Onboard/ChooseConnector/index.tsx:8]: hmmm. seems you're deleting the umbrel connector here?

**Comment by bumi** [comment]
closing in favor of #691

File: src/app/screens/Onboard/ChooseConnector/index.tsx (hunk 2 above)
Comment (secondl1ght) [reply src/app/screens/Onboard/ChooseConnector/index.tsx:8]: I can add this back in, wasn't sure if it would allow me to merge and resolve the conflicts.
